src/open_inwoner/configurations/management/commands/create_docs.py

```diff
@@ -0,0 +1,90 @@
+import os
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+
+from jinja2 import Environment, FileSystemLoader
+
+from open_inwoner.configurations.bootstrap.models import (
+    DigiDOIDCConfigurationSettings,
+    KICConfigurationSettings,
+    SiteConfigurationSettings,
+    ZGWConfigurationSettings,
+    eHerkenningDOIDCConfigurationSettings,
+)
+
+SUPPORTED_OPTIONS = ["siteconfig", "kic", "zgw", "digid_oidc", "eherkenning_oidc"]
+
+TEMPLATE_DIR = (
+    Path(os.path.abspath(os.path.dirname(__file__))).parent.parent
+    / "bootstrap"
+    / "templates"
+)
+TARGET_DIR = (
+    Path(os.path.abspath(os.path.dirname(__file__))).parent.parent.parent.parent.parent
+    / "docs"
+    / "configuration"
+)
```

Comment:
I'm really curious about the thought process here. Surely we can do better then `.parent.parent.parent.parent.parent`? Where does that even go?

Start from `settings.BASE_DIR` if the target is not in immediate vicinity of the local file.